Makefile

```diff
@@ -147,19 +149,16 @@ dist/.openapi-lint: internal/docs/swagger.yaml | dist
 		validate -i /src/internal/docs/swagger.yaml
 	$(CMD_PREFIX) touch $@
 
-dist/.gen-docs: $(APISERVER_DEPS) | dist
+$(SWAGGER_YAML): $(APISERVER_DEPS) | dist
 	$(ECHO_PREFIX) printf "  %-12s ./cmd/apiserver/main.go\n" "[API DOCS]"
 	$(CMD_PREFIX) docker run \--platform linux/x86_64 --rm \
 		-v $(CURDIR):/workdir -w /workdir \
 		ghcr.io/swaggo/swag:v1.8.10 \
 		/root/swag init $(SWAG_ARGS) -g ./cmd/apiserver/main.go -o ./internal/docs
-	$(CMD_PREFIX) touch $@
-
-internal/docs/swagger.yaml: dist/.gen-docs
 
 .PHONY: gen-openapi-client
-gen-openapi-client: dist/.gen-openapi-client ## Generate the OpenAPI Client
-dist/.gen-openapi-client: internal/docs/swagger.yaml | dist
+gen-openapi-client: internal/api/public/client.go ## Generate the OpenAPI Client
+internal/api/public/client.go: internal/docs/swagger.yaml | dist
```

Comment:
@chirino do you know if this is sufficient? In other words, if swagger.yaml got updated, it would always re-write client.go?

Reply:
seems right